MetaMask/metamask-mobile
# [FIX] Update assets-controllers patch

**Description**
This PR update the patch of `assets-controller` that was missing on this [PR](https://github.com/MetaMask/metamask-mobile/pull/5529)

**Test steps**
* Make Account#1 active on the wallet
* Connect with Account#2 to Uniswap
* Swap with a token that you do not currently own
* After the swap, you should be prompted with the ability to add the token to Account#2
* Proceed with adding the asset to Account#2
* Navigate to the Wallet tab
* Notice that the added asset does not show in Account#1
* Switch the wallet account to Account#2
* Notice the added asset appears in the asset list

**Screenshots/Recordings**
https://recordit.co/OQ3KOxtt4w

**Issue**

Progresses #https://github.com/MetaMask/mobile-planning/issues/686

**Checklist**
